### PR TITLE
Activate azure tests

### PR DESCRIPTION
**Pull request by mdellweg**
[noissue]

### CODE REVIEW

File: .github/workflows/scripts/post_before_script.sh [hunk 1]
```diff
@@ -1,10 +1,10 @@
 set -euv
 
-if [ "$TEST" = "pulp" ] || [ "$TEST" = "performance" ] || [ "$TEST" = "s3" ]
+if [ "$TEST" = "pulp" ] || [ "$TEST" = "performance" ] || [ "$TEST" = "azure" ] || [ "$TEST" = "s3" ]
 then
   # Add signing service script and setup script:
-  cat pulp_deb/tests/functional/sign_deb_release.sh | cmd_stdin_prefix bash -c "cat > /root/sign_deb_release.sh"
-  cat pulp_deb/tests/functional/setup_signing_service.py | cmd_stdin_prefix bash -c "cat > /tmp/setup_signing_service.py"
+  cmd_stdin_prefix bash -c "cat > /root/sign_deb_release.sh" < pulp_deb/tests/functional/sign_deb_release.sh
+  cmd_stdin_prefix bash -c "cat > /tmp/setup_signing_service.py" < pulp_deb/tests/functional/setup_signing_service.py
```
Comment (quba42) [inline .github/workflows/scripts/post_before_script.sh:7]: For future reference, why is `<` preferable to `cat |`?

**Review by quba42 — APPROVED** [review]

File: .github/workflows/scripts/post_before_script.sh (hunk 1 above)
Comment (mdellweg) [reply .github/workflows/scripts/post_before_script.sh:7]: It's one process less to start up from the shell.

Comment (quba42) [reply .github/workflows/scripts/post_before_script.sh:7]: Shellcheck says: "useless use of cat"

Comment (mdellweg) [reply .github/workflows/scripts/post_before_script.sh:7]: The one thing i don't like about it is that the beginning of the pipe is moved to the end of the line.

**Pull request merged**